addons/l10n_gcc_pos/static/src/js/OrderReceipt.js

```diff
@@ -0,0 +1,20 @@
+odoo.define('l10n_gcc_pos.OrderReceipt', function (require) {
+    'use strict';
+
+    const OrderReceipt = require('point_of_sale.OrderReceipt')
+    const Registries = require('point_of_sale.Registries');
+
+    const OrderReceiptGCC = OrderReceipt =>
+        class extends OrderReceipt {
+
+            get receiptEnv() {
+                let receipt_render_env = super.receiptEnv;
+                let receipt = receipt_render_env.receipt;
+                receipt.is_gcc_country = ['SA', 'AE', 'BH', 'OM', 'QA', 'KW'].includes(receipt_render_env.order.pos.company.country.code);
```

Comment:
Hard-coding the list seems less than ideal as the membership can change over time, at least in theory: according to the Wiki, Yemen has been trying to join for a while (though the ongoing crisis has obviously killed that.

Reply:
I think it is quite easy to update the list, as it is only set at one place.